### ISSUE TITLE
Make refresh rate configurable in CLI

### ISSUE THREAD

**Issue by aodhan-domhnaill** [medic/couch2pg#29]
https://github.com/medic/couch2pg/blob/871d4c96cb82dc17de80b3745c9a53d966e8294b/env.js#L47

It would be useful to be able to configure the rate that couch2pg refreshes so that local development can refresh faster.

**Comment by garethbowen** [comment]
Agreed! Feel free to submit a PR for this with documentation. I propose we release it but don't worry about updating any prod instances as they won't be setting it anyway.
